src/main/java/com/github/cypher/sdk/Message.java

```diff
@@ -0,0 +1,22 @@
+package com.github.cypher.sdk;
+
+import com.github.cypher.sdk.api.ApiLayer;
+import com.google.gson.JsonObject;
+
+public class Message extends Event {
+	private String body = "";
```

Comment:
I left it as mutable because of the possibility that Matrix might enable editing of messages in the future.
Although I haven't given any thought as to haw we might implement that.